docs/csv.md

```diff
@@ -0,0 +1,115 @@
+# CSV import
+
+`reccmp` has the option to parse [comma-separated value (CSV)](https://en.wikipedia.org/wiki/Comma-separated_values) files and load data to supplement code annotations on the original binary.
+
+Any metadata set from a CSV file will overwrite the same data from code annotations on the same address.
+
+If the same address is repeated on multiple lines the same CSV file, or across multiple files, the metadata will be overwritten each time.
+
+## Setup
+
+Add the list of CSV files under the `data-sources` key for the target in `reccmp-project.yml`. For example:
+
+```yml
+targets:
+  ISLE:
+    filename: ISLE.EXE
+    source-root: .
+    hash:
+      sha256: 5cf57c284973fce9d14f5677a2e4435fd989c5e938970764d00c8932ed5128ca
+    data-sources:
+      - test.csv
+```
+
+The paths are relative to the directory that contains `reccmp-project.yml`.
+
+## Format
+
+Here's an example of a CSV file using the pipe character[^1] as the delimiter.
```

Comment:
This footnote syntax does not appear to work in VS Code, though it's fine in GitHub. Are you aware of a more compatible format? If not, I'd leave it like that.